tools/droplets/fabric/fabfile.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
```

Comment:
Is this code unmodified from upstream?

Reply:
@timabbott Nope. I made changes like
1) Adding setup_interactive_login_script function
2) Fixing the lint errors
3) Removed some unnecessary comments and unused functions.
4) Adding mypy
5) Not using `from fabric.api import *`
6) Moved the print syntax to Python3

This is the actual file https://github.com/digitalocean/marketplace-partners/blob/master/marketplace_docs/template/fabfile.py